tests/bluetooth/tester/overlay-mesh.conf

```diff
@@ -19,9 +19,9 @@ CONFIG_BT_MESH_CFG_CLI=y
 CONFIG_BT_MESH_HEALTH_CLI=y
 CONFIG_BT_MESH_FRIEND=y
 CONFIG_BT_MESH_FRIEND_QUEUE_SIZE=32
-CONFIG_BT_MESH_RX_SEG_MAX=13
-CONFIG_BT_MESH_TX_SEG_MAX=8
-CONFIG_BT_MESH_TX_SEG_MSG_COUNT=3
+CONFIG_BT_MESH_RX_SEG_MAX=14
+CONFIG_BT_MESH_TX_SEG_MAX=13
+CONFIG_BT_MESH_TX_SEG_MSG_COUNT=10
```

Comment:
Ohhhh. I didn't know this.

We should set TX/RX SEG_MAX to 32, even though we may not be using them in the tester, to ensure we don't have to keep updating them every now and then.